### DISCUSSION TITLE
Plots K33 k34 and k35

### DISCUSSION THREAD

**Discussion by rotsencrypto** [madMAx43v3r/chia-plotter#754] [Q&A]
how can I create plots k 33 or 34 using mad max

**Comment by SebMoore** [comment]
> how can I create plots k 33 or 34 using mad max

At the moment, you cannot.

**Reply by gorbi13G** [reply]
feature request - add please K33 / K34 / K35 :-)

**Comment by cjedwards** [comment]
Any chance on k33+ support?
I wouldn't mind being able to better optimize my HDD space using different K values !

Cheers

**Reply by gryan315** [reply]
This is a ridiculous concept and gains you almost nothing. If you actually want better chances to win, create a logical volume across your leftover space and fit an actual extra k32 plot. If one drive fails you lose 1 extra plot.

**Reply by gorbi13G** [reply]
when you are combining K32 with K33 you can always have less than 7GB space left ( K33 - 2x K32 )
If you have lot of K32 and K33 you can choose bigger and smaller to create the best combination to have almost 0bytes left free on the storage. Additional Logical volume is waste of space.. ;p but also can be a solution.. it depends..

I have here some examples:

18TB  118*K32 +  23*K33 (3,2GiB free left)
14TB  40*K32 + 43*K33  (3GiB free left)
12TB  67*K32 + 21*K33 (almost zero 0,2GiB free left)
10TB  28 + 31
8TB    22 + 25 
4TB   12 + 12
2TB    6 + 6

i have created google sheet with some math to get some cobinations... to have less than 7GB of free space left

**Reply by cjedwards** [reply]
Seems you misunderstand gryan,
This has nothing to do with "better chances to win" this is about optimizing space for a space/time application!
as gorbi13G has shown it makes A LOT of sense to combine the types to MAximize space.

**Reply by altendky** [reply]
Keep in mind that on the 18TB that's about a 0.2% increase in capacity.  If we take a nominal $500 for the disk, that's $1 wasted.  Yes, it is more on the smaller disks, but if we are concerned about efficiency then the tiny disks are not a great choice anyways.  If we take a 10TB and assume worst/best cases then we have 1% improved space usage.  I personally have decided that having to adapt plotting optimizations to various sizes and having to track various size plots is just not that valuable.  I could instead skip that and attach a single extra small cheap disk to my farm and get that extra capacity that way for negligible cost and effort.

But sure, maybe someday (ten years?) we'll want k33 for the network.

**Reply by cjedwards** [reply]
k32 will be dead in less than 2 years. Never mind the original thought of 10 years, Even the Chia team will not comment any more on how fast k33 will be needed, as every time they do they realize they were wrong..

It is always better to be proactive than reactive!

**Reply by gorbi13G** [reply]
in future we can offcourse use another combinations to maximize disk space usage.. always we will have less then
(K34 - 2xK33)
or
(K35 - 2xK34)
left free space.

The best will be if GPU can directly generate plot K32 / K33 / K34 / K35 directly to disk :-)

in BURST / BHD / LHD etc was very easy to use GPU plot generator v4.2.2.. yes.. chia is a little bit more complicated to generate it in many phases and required many temporary disk space..

**Reply by gryan315** [reply]
> Seems you misunderstand gryan,
> This has nothing to do with "better chances to win" this is about optimizing space for a space/time application!
> as gorbi13G has shown it makes A LOT of sense to combine the types to MAximize space.

I was not misunderstanding, I just believe that the logic behind this strategy is flawed, and that you will not see any actual benefit from doing this. If you instead spanned a logical volume across some of your free space (enough to add one extra k32) you would likely see more real world benfit.

**Comment by SebMoore** [comment]
#25, #65, #217, #416, #478. Not yet. Will require large rewrite of plotter, as plot size is a hardcoded value - not something easily changed.

**Reply by cjedwards** [reply]
A shame!
I would love to stop playing with k32 at all. and go right to k33 k34
I am not looking forward to next year replotting anyway when the Chia team realizes k32 was too small

**Reply by gryan315** [reply]
> A shame!
> I would love to stop playing with k32 at all. and go right to k33 k34
> I am not looking forward to next year replotting anyway when the Chia team realizes k32 was too small

This seems fairly unlikely.

**Reply by cjedwards** [reply]
> > A shame!
> > I would love to stop playing with k32 at all. and go right to k33 k34
> > I am not looking forward to next year replotting anyway when the Chia team realizes k32 was too small
> 
> This seems fairly unlikely.

Unless you are on the Chia team, what you "seem" is irrelevant if I want to plot bigger plots :/
